docs/networks/l16-testnet.md

```diff
@@ -194,11 +188,11 @@ make submit-deposit
 # wait 8h till validator is activated
 ```
 
-You will need to wait for 8 hours to activate your validator.
+You will need to wait for eight hours to activate your validator.
 
 ### Run the validator client
 
-Once your validator is activated you spin up a validator client. **Make sure your _consensus_ and _execution_ clients are running (running `make start`).**
+Once your validator is activated, you spin up a validator client. <br></br>**Make sure your _consensus_ and _execution_ clients are running (by typing `make start`).**
```

Comment:
```suggestion
Once your validator is activated, you spin up a validator client.

**Make sure your _consensus_ and _execution_ clients are running (by typing `make start`).**
```

avoid using `<br>` - let's markdown handle this for us. If you need line return, add 2 line retun.

(also br has no closing tag, `</br>` is not valid)